src/components/forms/search-user-form/search-user-form.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Form, Header, Message, Segment } from 'semantic-ui-react';
+import { Input } from 'components/forms';
+import { reduxForm, Field } from 'redux-form';
+import { FORM_ERROR_REQUIRED_FIELD } from 'components/forms/errors';
+
+const validate = (values, props) => {
+  const errors = {};
+
+  if (!values.username || values.username.length === 0) {
+    errors.username = FORM_ERROR_REQUIRED_FIELD;
+  }
+
+  if (!props.submitting && (!values.bilingId || (values.bilingId && values.bilingId.length === 0))) {
+    errors.bilingId = FORM_ERROR_REQUIRED_FIELD;
+  }
+
+  return errors;
+};
```

Comment:
if we search a user, do we realy want to submit that earch? shouldn't it be dynamic onChange?